### PR TITLE
add actionables documentation

### PR DESCRIPTION
**Pull request by djfritz**

### CODE REVIEW

File: gui/actionables/actionables.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Actionables
+
+## Overview
+
+Actionables provide a way to create custom triggers and menus that key on any text rendered in a query and take one or more actions when selected. Similar to an HTML Hyperlink, actionables can be used to open external URLs that key on data, but actionables can also be leveraged to submit new Gravwell queries, launch dashboards, and execute templates.
+
+Actionables are created by specifying one or more regular expressions, along with one or more actions. Gravwell automatically parses all text rendered with the [table](#!/search/table/table.md) and [chart](#!/search/chart/chart.md) renderers and creates menus for any provided triggers.
```
Comment (traetox) [inline gui/actionables/actionables.md:7]: i don't believe a single actionable can have more than one regex

File: gui/actionables/actionables.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+The URL action supports opening a new window/tab with a given URL and matched text, and additionally provides a set of timestamp options for providing the time range arguments from the search that the actionable triggered on. 
+
+The "Open in a modal" option opens the URL in a window within the current Gravwell instance, similar to an HTML iframe.
+
+#### Run a Saved Query
+
+Simply run a query from the Query Library. 
+
+
```
Comment (traetox) [inline gui/actionables/actionables.md:108]: Add a section about names and grouping in that menu?  Maybe show one with overlapping names in the menu vs. not?

**Review by traetox — CHANGES_REQUESTED** [review]

File: gui/actionables/actionables.md (hunk 1 above)
Comment (djfritz) [reply gui/actionables/actionables.md:7]: it can.

**Review by traetox — APPROVED** [review]

File: index.md [hunk 3]
```diff
@@ -108,7 +108,9 @@ Gravwell is pleased to announce our free [Community Edition](https://www.gravwel
 
     * [Labels and Filtering](gui/labels/labels.md)
 
-	* [Kits](kits/kits.md)
+    * [Kits](kits/kits.md)
+
+    * [Actionables](gui/actionables/actionables.md)
```
Comment (floren) [inline index.md:113]: I try to keep the index small by only having really big stuff on index.md, and I think actionables may be a bit advanced for it?

**Comment by djfritz** [comment]
@floren PTAL

**Review by floren — APPROVED** [review]

**Pull request merged**